pay/payment-providers/EFT.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Betalen</title>
+    </head>
```

Comment:
Als je een component maakt moet je alleen de html van het daadwerkelijke component inladen. een head en body zijn dus overbodig want die zijn er al. Het zou dus zo moeten worden:

<form method='get' action='betaalpagina.php'>
    Maak het totaalbedrag binnen 2 werkdagen over naar IBAN:<br>
    <br>
    NL12 TEST 1234 5678 90<br>
    <br>
    Vul bij het betalingskenmerk het ordernummer in, die u op de factuur krijgt...<br>
    <br>
    <input type='submit' value='Bestellen'>
</form>

Reply:
Got it, resolved